ui-components/src/actions/ExpandableText.tsx

```diff
@@ -29,10 +30,16 @@ const getText = (text: string, expanded: boolean, maxLength: number) => {
 const moreLessButton = (
   expanded: boolean,
   setExpanded: (newValue: boolean) => void,
-  strings: ExpandableTextProps["strings"]
+  strings: ExpandableTextProps["strings"],
+  buttonClassName: ExpandableTextProps["buttonClassName"]
 ) => {
+  const classes = ["button-toggle"]
```

Comment:
The `ml-4` looks like a great bug fix for us as well, do you mind adding that to the button-toggle class?

Reply:
@emilyjablonski pushed the fix!